ephios/static/ephios/scss/ephios_custom.scss

```diff
@@ -266,3 +266,8 @@ a.badge {
 .notification-content ul {
   padding-bottom: 1rem;
 }
+
+.unread-notification {
+  border-left-color: $primary;
+  border-left-width: 0.5rem;
+}
```

Comment:
Not bad. I think it should be
```suggestion

.notification {
  border-left-width: 0.5rem;
  border-left-color: transparent;
}

.notification-unread {
  border-left-color: $primary;
}
```
or something like that so everything stays neatly left-aligned...

Reply:
this does not work because the list-group-item is already using a slim border, so making it transparent looks ugly

Reply:
My point is: make it left-aligned ^^
![image](https://github.com/ephios-dev/ephios/assets/5970076/d6c1d3c6-1495-4d72-9f27-fc7a7ed5b456)